packages/playground/wordpress/src/index.ts

```diff
@@ -171,10 +171,16 @@ export async function setupPlatformLevelMuPlugins(php: UniversalPHP) {
 				$parts = explode('/', $redirect_url);
 				$redirect_url = '/' . implode('/', array_slice($parts, 2));
 			}
-			wp_redirect(
-				home_url($redirect_url),
-				302
-			);
+			$redirect_url = home_url($redirect_url);
```

Comment:
@adamziel we don't need to force this to a absolute URL anymore because we [have a a fix for the relative URL redirects in Safari.](https://github.com/WordPress/wordpress-playground/pull/1978)
```suggestion
```

Reply:
I hope that you don't mind, that I merged trunk into your PR and resolved the conflict with the[ Safari redirect fix](https://github.com/WordPress/wordpress-playground/pull/1978).